### PR TITLE
Enter jam edit

### PR DESCRIPTION
**Pull request by indiegamehunt**

### CODE REVIEW

File: src/enter-jam/index.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Making a game page before your game is finished allows people to follow its deve
 
 To enter your game in the jam, **put the jam hashtag in the game's description**. You can edit the description from the game's management page on your [dashboard](http://gamejolt.com/dashboard/). Make sure to add the hashtag after the jam begins, but before the jam ends.
 
+Note that adding a jam hashtag creates a jam page for your game that is different from the original game page.
```
Comment (hworld) [inline src/enter-jam/index.md]: Maybe "adding a jam hashtag creates an entry page on the jam site" instead? Or similar?

File: src/enter-jam/index.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ Making a game page before your game is finished allows people to follow its deve
 
 To enter your game in the jam, **put the jam hashtag in the game's description**. You can edit the description from the game's management page on your [dashboard](http://gamejolt.com/dashboard/). Make sure to add the hashtag after the jam begins, but before the jam ends.
 
+Note that adding a jam hashtag creates an entry page on the jam site that is different from the original game page.
```
Comment (MaskofThoro) [inline src/enter-jam/index.md]: What do you think of: 

"Note that adding a jam hashtag creates a **separate** entry page on the jam site that is different from the original game page." 

also should we add something like "Voting will be calculated based on the ratings from the jam entry page, not the original game page"

Comment (hworld) [inline src/enter-jam/index.md]: The part I don't like is actually "different from the original game page". I think it's enough to say it'll create a separate page for their jam entry on the jam site.

Comment (MaskofThoro) [inline src/enter-jam/index.md]: sounds good to me. how do you feel about adding the voting thing?

Comment (hworld) [inline src/enter-jam/index.md]: Hmm, sure. What would you suggest?

Comment (MaskofThoro) [inline src/enter-jam/index.md]: I suggest that we add it because it has been a pressure point for confusion in the past.

Comment (indiegamehunt) [inline src/enter-jam/index.md]: Yeah, specifying that voting happens on that page helps clarify and explain that there are separate pages. See if this  latest works.

**Comment by indiegamehunt** [comment]
Shouldn't this be closed?

**Comment by indiegamehunt** [comment]
Updated to reference releases, etc. for the new site. Ready for review.

**Pull request merged**